dev.neovide.neovide.yml

```diff
@@ -0,0 +1,53 @@
+id: dev.neovide.neovide
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# sdk-extensions:
+#   - org.freedesktop.Sdk.Extension.rust-stable
+command: neovide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri # opengl
+  - --share=network
+  - --share=ipc
+  - --filesystem=host
+  - --filesystem=xdg-config/neovide
+  - --talk-name=org.freedesktop.Flatpak
+modules:
+  - name: neovide
+    sources:
+      # - type: archive
+      #   url: https://github.com/neovide/neovide/archive/refs/tags/0.12.2.tar.gz
+      #   sha256: 5425c60454388651fd79757bde7c4d7499cdc49b375f7697b48d8366d45d08e4
+      # - generated-sources.json
+      - type: git
+        url: https://github.com/sqwxl/neovide.git
+        commit: 18352f8649c4c5b36665d1041debee9be2fcd40f
+        dest: repo
+      - type: file
+        url: https://github.com/sqwxl/neovide/releases/download/0.12.3/neovide-linux-x86_64.tar.gz
+        sha256: 2307e7c9599f3e32dfbd61054e49ed1c41945a399286a75b52a5d3bc0bdd021a
+      - type: file
+        path: dev.neovide.neovide.metainfo.xml
+    buildsystem: simple
+    # build-options:
+      # append-path: /usr/lib/sdk/rust-stable/bin
+      # env:
+      #   CARGO_HOME: /run/build/neovide/cargo
+    build-commands:
+      # - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      # - cargo --offline build --release --verbose
+      - tar xzf neovide-linux-x86_64.tar.gz
+      - install -Dm755 neovide -t /app/bin/
+
+      - install -Dm644 repo/assets/neovide-16x16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png
+      - install -Dm644 repo/assets/neovide-32x32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png
+      - install -Dm644 repo/assets/neovide-48x48.png /app/share/icons/hicolor/48x48/apps/${FLATPAK_ID}.png
+      - install -Dm644 repo/assets/neovide-256x256.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png
```

Comment:
Do you actually need the smaller icons? If not the SVG is enough, or keep 256px and the svg.